diff --git a/node_modules/react-native/sdks/hermes-engine/utils/replace_hermes_version.js b/node_modules/react-native/sdks/hermes-engine/utils/replace_hermes_version.js
index f727aeb..49bceae 100644
--- a/node_modules/react-native/sdks/hermes-engine/utils/replace_hermes_version.js
+++ b/node_modules/react-native/sdks/hermes-engine/utils/replace_hermes_version.js
@@ -61,7 +61,7 @@ function replaceHermesConfiguration(configuration, version, podsRoot) {
   fs.mkdirSync(finalLocation, {recursive: true});
 
   console.log('Extracting the tarball');
-  execSync(`tar -xf ${tarballURLPath} -C ${finalLocation}`);
+  execSync(`tar -xf "${tarballURLPath}" -C "${finalLocation}"`);
 }
 
 function updateLastBuildConfiguration(configuration) {
